### PR TITLE
Shivam/upgrade to 3.3.x

### PR DESCRIPTION
**Pull request by shivamkrpandey**

### CODE REVIEW

**Comment by shivamkrpandey** [comment]
[(4) SPYD - Upgrade mongeez plugin to 3.3.x](https://trello.com/c/VsQbnRTx/510-4-spyd-upgrade-mongeez-plugin-to-33x)

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Grails MongoDB Database Migration Plugin
-Latest Version (0.2.4)
+Latest Version (0.2.5)
```
Comment (ankit-agrawal11) [inline README.md]: major release. Please update the version accordingly.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 ## Overview 
 
-This MongoDB database migration plugin helps to manage database changes while developing Grails 3.2.x Applications with MongoDB.
+This MongoDB database migration plugin helps to manage database changes while developing Grails 3.3.x and 3.2.x Applications with MongoDB.
```
Comment (ankit-agrawal11) [inline README.md:6]: Please mention the versions supported for respective Grails versions.

File: build.gradle [hunk 3]
```diff
@@ -33,7 +33,7 @@ buildscript {
     }
 }
 
-version "0.2.4"
+version "0.2.5"
```
Comment (ankit-agrawal11) [inline build.gradle]: Upgrade version.

File: build.gradle [hunk 4]
```diff
@@ -126,3 +131,7 @@ grailsPublish {
     desc = "Grails Mongeez plugin for Grails 3"
     developers = [causecode:"CauseCode Technologies Pvt Ltd"]
 }
+
+task wrapper(type: Wrapper) {
```
Comment (ankit-agrawal11) [inline build.gradle:135]: Please upgrade the wrappers by running the task.

File: README.md [hunk 5]
```diff
@@ -12,11 +12,12 @@ This plugin uses the [Mongeez](https://github.com/mongeez/mongeez) library and c
 ## Installation
  * First add a dependency for the plugin in build.gradle
 
+Grails App >= 3.3.x use `v0.2.5` and <=3.2.x use `v0.2.4`
 ```
     buildscript {
         dependencies {
             ...
-            classpath "com.causecode.plugins:grails-mongeez:0.2.4"
+            classpath "com.causecode.plugins:grails-mongeez:$version"
```
Comment (ankit-agrawal11) [inline README.md:20]: Please update changelog. If there is none, please add one.

**Review by ankit-agrawal11 — CHANGES_REQUESTED** [review]

**Review by hardikmodha — CHANGES_REQUESTED** [review]
Make sure you have tested the migrations with at least one test app.

File: README.md (hunk 1 above)
Comment (shivamkrpandey) [reply README.md]: Ok Changing it to 3.3.0 and before final review, let's discuss with @visheshd for versioning in all the plugins.

File: README.md (hunk 2 above)
Comment (shivamkrpandey) [reply README.md:6]: It supports in all 3.2.x and 3.3.x so not better to explicitly mention grails version.

File: build.gradle (hunk 3 above)
Comment (shivamkrpandey) [reply build.gradle]: Done

File: build.gradle (hunk 4 above)
Comment (shivamkrpandey) [reply build.gradle:135]: Ok

**Comment by shivamkrpandey** [comment]
@HardikModha Tested with an app (+ migrations)

File: README.md (hunk 5 above)
Comment (shivamkrpandey) [reply README.md:20]: Done, it was not present.

File: changelog.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# ChangeLog
+
+## [1.0.0] - [Unreleased]
+### Changed
+- Upgraded the plugin to support grails 3.3.5
```
Comment (ankit-agrawal11) [inline changelog.md]: 3.3.x

**Review by hardikmodha — APPROVED** [review]

**Review by ankit-agrawal11 — APPROVED** [review]

**Pull request merged**